actionmailbox/test/controllers/rails/action_mailbox/inbound_emails_controller_test.rb

```diff
@@ -45,8 +45,10 @@ class Rails::Conductor::ActionMailbox::InboundEmailsControllerTest < ActionDispa
       end
 
       mail = ActionMailbox::InboundEmail.last.mail
+      attachment_filenames = mail.attachments.map(&:filename)
       assert_equal "Let's talk about these images:", mail.text_part.decoded
       assert_equal 2, mail.attachments.count
+      assert_equal attachment_filenames, ["avatar1.jpeg", "avatar2.jpeg"]
```

Comment:
Shouldn't this assertion be like `assert_equal expected, actual`. So in this assertion, it is should be as follows:

```
assert_equal ["avatar1.jpeg", "avatar2.jpeg"], attachment_filenames
```

Reply:
Ahh yes - good catch - would you like me to submit a new PR since this one has already been merged?

Reply:
This seems to be already fixed in https://github.com/rails/rails/commit/72bc0806a7b378cd544e8fbf7ab22d74b7913ffb